Formula/eksctl.rb

```diff
@@ -0,0 +1,35 @@
+class Eksctl < Formula
+  desc "The official CLI for Amazon EKS"
+  homepage "https://eksctl.io"
+  url "https://github.com/weaveworks/eksctl.git",
+      :tag      => "0.24.0",
+      :revision => "f23f8d7d75943d1a62c87ce2f3d455da7af741fa"
+  license "Apache-2.0"
+
+  depends_on "go" => :build
+  depends_on "go-bindata" => :build
+  depends_on "aws-iam-authenticator"
+  depends_on "kubernetes-cli"
```

Comment:
This might be our value proposition.

But we probably dont have to specify the k8s dependency though.

Reply:
I am with you on this one. I removed the dependency on `kubernetes-cli`

Reply:
This is something I did before to migrate a tap formula to homebrew-core, `https://github.com/loadimpact/homebrew-k6/pull/7`, you might help it helpful.